test/wasmfs/get_backend.h

```diff
@@ -13,6 +13,10 @@
 
 #include <emscripten/wasmfs.h>
 
+#ifdef __cplusplus
+using namespace wasmfs;
+#endif
```

Comment:
(This concerns me as well, for the above reasons about namespacing.)

Reply:
I agree - I think going back to forward declarations should resolve this issue.